### PR TITLE
fix(worker): Avoid throwing error when there's no activities directory

### PR DESCRIPTION
**Pull request by vkarpov15**
## What was changed

Added a missing `mkdir()` call

## Why?

If there's no activities, `registerActivities()` doesn't call `mkdir()`, so the `writeFileSync()` call will fail

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Review by bergundy — APPROVED** [review]
Thanks!

**Comment by joebowbeer** [comment]
I'm still seeing a residual error in samples-node/progress:

> Error: ENOENT: no such file or directory, open '/src/main.js'

File: packages/worker/src/isolate-builder.ts [hunk 1]
```diff
@@ -98,6 +98,11 @@ export class WorkflowIsolateBuilder {
 
       module.exports = api;
     `;
+    try {
+      vol.mkdirSync(path.dirname(target), { recursive: true });
+    } catch (err) {
+      if (err.code !== 'EEXIST') throw err;
+    }
     vol.writeFileSync(target, code);
```
Comment (joebowbeer) [inline packages/worker/src/isolate-builder.ts:106]: It will still blow up here, right?

**Comment by bergundy** [comment]
@joebowbeer we still haven't released the fix which would explain why you're still seeing the error.
`genEntrypoint` is called only once and AFAICT @vkarpov15's fix addresses the issue.

**Comment by joebowbeer** [comment]
@bergundy thanks for explanation. I see now: the unreleased fix is creating the dir if it doesn't exist.

**Pull request merged**